packages/mobile/src/firebase/dynamicLinks.ts

```diff
@@ -0,0 +1,36 @@
+import firebase from 'react-native-firebase'
+import Logger from 'src/utils/Logger'
+
+const TAG = 'firebase/dynamicLink'
+
+export async function generateShortInviteLink({
+  link,
+  playStoreUrl,
+  appStoreUrl,
+  bundleId,
+}: {
+  link: string
+  playStoreUrl: string
+  appStoreUrl: string
+  bundleId: string
+}): Promise<string> {
+  try {
+    const firebaseLink = new firebase.links.DynamicLink(link, 'https://celol.page.link')
```

Comment:
Is celo**l** intentional?

Reply:
Yes, because `celo.page.link` is used in other firebase project according to @martinvol. But anyway this is temporary - we would use our own domain for this (I just need to figure out who can add TXT DNS record to our domain).